### PR TITLE
Bug 1600626 - Add progress status to Push Health

### PR DESCRIPTION
**Pull request by camd**

### CODE REVIEW

File: treeherder/model/models.py [hunk 1]
```diff
@@ -147,12 +147,13 @@ def get_status(self):
             Q(failure_classification__isnull=True) |
             Q(failure_classification__name='not classified')).exclude(tier=3)
 
-        status_dict = {}
+        status_dict = {'completed': 0, 'pending': 0, 'running': 0}
```
Comment (camd) [inline treeherder/model/models.py]: Ensure these states exist whether 0 or otherwise.

File: ui/push-health/Navigation.jsx [hunk 2]
```diff
@@ -3,30 +3,34 @@ import PropTypes from 'prop-types';
...
-            <div>
-              This prototype is still in-progress. The section on `Tests` has
-              been implemented, but Linting, Coverage, Builds, etc are not. You
-              will notice in Treeherder that the status may say `OK` for a push
-              that has failed Builds or linting. These features will be updated
-              in the weeks to come.
-            </div>
-          }
-        />
+        <h4>
```
Comment (camd) [inline ui/push-health/Navigation.jsx]: This link seemed better in the nav bar.  It looked less busy this way, somehow.  We also don't need the "Prototype" info anymore.

File: ui/push-health/Navigation.jsx [hunk 3]
```diff
@@ -3,30 +3,34 @@ import PropTypes from 'prop-types';
...
+          <span className={`badge badge-${overallResult}`}>
```
Comment (sarah-clements) [inline ui/push-health/Navigation.jsx]: Does the reactstrap Badge component work here?

Comment (camd) [reply ui/push-health/Navigation.jsx]: Yep, right again.  :). Fixed.

**Pull request merged**